### PR TITLE
Update Core Doc Text

### PR DESCRIPTION
**Pull request by electrostat**
- update the text (snippets already updated) for the Core Docs to bring them inline with `Core 1.1.0`

cc: @andrlee

### CODE REVIEW

**Comment by langsmith** [comment]
screencapture-localhost-8080-android-docs-core-overview-2018-12-11-13_59_41

File: src/pages/core/overview/index.md [hunk 1]
```diff
@@ -174,40 +174,47 @@ This will obtain the best location engine that is available and eliminate the ne
 
 ### Getting location updates
 
-Add the `mapbox-android-core` dependency to your project in order to listen to location changes. Then initialize a new instance of `LocationEngine` as described above. Once it's created, activate the `LocationEngine` object and optionally add a location listener. Implementing the `LocationEngineListener` interface will require you to override the `onConnected()` and `onLocationChanged()` methods. Inside of `onConnected()`, you can begin requesting location updates or wait for the proper time to do so.
+Add the `mapbox-android-core` dependency to your project in order to listen to location changes. Then initialize a new instance of `LocationEngine` as described above. Once it's created, activate the `LocationEngine` object. Either the parent Acticity should implement `LocationEngineCallback` or a separate `LocationEngineCallback` object to include in your update request. Implementing the `LocationEngineCallback` interface will require you to override the `onSuccess()` and `onFailure()` methods. Inside of `onSuccess()`, you will receive a `LocationEngineResult` that contains location results from the engine.
```
Comment (langsmith) [inline src/pages/core/overview/index.md]: `Acticity` 👉 `Activity`

Comment (langsmith) [reply src/pages/core/overview/index.md]: nit `that`>`which`


`Inside of `onSuccess()`, you will receive a `LocationEngineResult` that contains location results from the engine.` 👉 `Inside of `onSuccess()`, you will receive a `LocationEngineResult` which contains location results from the engine.`

Comment (electrostat) [reply src/pages/core/overview/index.md]: good find

**Pull request merged**